src/org/labkey/targetedms/outliers/CUSUMOutliers.java

```diff
@@ -111,42 +118,46 @@ private String getEachSeriesTypePlotDataSql(String type, int id, String schemaNa
 
     private String queryContainerSampleFileRawData(List<QCMetricConfiguration> configurations)
     {
-        StringBuilder sqlBuilder = new StringBuilder();
+        SQLFragment sqlBuilder = new SQLFragment();
         String sep = "";
         String where ="";
 
+        sqlBuilder.append(addParametersToSQL(configurations));
+        sqlBuilder.append("SELECT * FROM (");
+
         for(QCMetricConfiguration configuration: configurations)
         {
             int id = configuration.getId();
             String label = configuration.getSeries1Label();
             String schema1Name = configuration.getSeries1SchemaName();
             String query1Name = configuration.getSeries1QueryName();
-            sqlBuilder.append(sep).append("(").append(getEachSeriesTypePlotDataSql("series1", id, schema1Name, query1Name, where, label)).append(")");
+            sqlBuilder.append(sep).append("(").append(getEachSeriesTypePlotDataSql("series1", id, schema1Name, query1Name, where, METRIC_LABEL_ONE)).append(")");
             sep = "\nUNION\n";
 
             if(configuration.getSeries2SchemaName() != null && configuration.getSeries2QueryName() != null) {
                 String schema2Name = configuration.getSeries2SchemaName();
                 String query2Name = configuration.getSeries2QueryName();
                 label = configuration.getSeries2Label();
-                sqlBuilder.append(sep).append("(").append(getEachSeriesTypePlotDataSql("series2", id, schema2Name, query2Name, where, label)).append(")");
+                sqlBuilder.append(sep).append("(").append(getEachSeriesTypePlotDataSql("series2", id, schema2Name, query2Name, where, METRIC_LABEL_TWO)).append(")");
             }
         }
-        return "SELECT * FROM (" + sqlBuilder.toString() + ") a"; //wrap unioned results in sql to support sorting
+        sqlBuilder.append(") a");
+        return  sqlBuilder.toDebugString() ; //wrap unioned results in sql to support sorting
```

Comment:
This method is for debugging purposes - probably want to use getSQL() or just return SQLFragment directly